Dockerfile

```diff
@@ -1,7 +1,8 @@
-FROM openjdk:8-jdk
+FROM openjdk:8-slim
 
-ENV FILE_NAME distZip
+ENV DIR_NAME Cosmas-1.0-SNAPSHOT
```

Comment:
Cool, this looks way better. The remaining issues:
1. The name of this variable is misleading. It is not a directory name, although it appears in the directory name by coincidence. It is rather cosmas version or something like that
2. It is needed at build time only (thsi value in the runtime is pretty useless), yet it is impossible to pass the value of this environment from the outside during build time. Instruction [`ARG`](https://docs.docker.com/engine/reference/builder/#arg) is designed for configuring dockerfile and build time.